pynicotine/slskproto.py

```diff
@@ -1029,6 +1048,11 @@ def _close_connection(self, connection_list, sock, callback=True):
         conn_type = init.conn_type
         user = init.target_user
 
+        if (conn_type == ConnectionType.DISTRIBUTED and self._child_peers.pop(user, None)
+                and len(self._child_peers) == self._max_distrib_children - 1):
+            log.add_conn("Accepting new distributed child peers")
```

Comment:
"Accepting new distributed child peers" -> "Available to adopt a new distributed child peer"